servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -753,7 +750,7 @@ boolean processUpdateTags(Map<String, TagValueList> currentTags, Map<String, Tag
         if (originalTags == null || originalTags.isEmpty()) {
             if (currentTags == null || currentTags.isEmpty()) {
                 // no tags to process..
-                return true;
+                return false;
```

Comment:
we can't just make this change. with the change, if there is nothing to process, it's a failure.